{
  "cjdns": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2339969"
    ],
    "summaries": [
      "cjdns: FTBFS in Fedora rawhide/f42"
    ]
  },
  "conda-build": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2356162"
    ],
    "summaries": [
      "conda-build fails to build with Python 3.14: TypeError: _WorkItem.__init__() takes 3 positional arguments but 5 were given"
    ]
  },
  "dolfin": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2371707"
    ],
    "summaries": [
      "F43FailsToInstall: python3-dolfin [retired on f44+, obsoleted on f43 by fedora-obsolete-packages]"
    ]
  },
  "matrix-synapse": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2385171",
      "https://bugzilla.redhat.com/show_bug.cgi?id=2371743"
    ],
    "summaries": [
      "matrix-synapse: FTBFS in Fedora rawhide/f43",
      "F43FailsToInstall: Multiple packages built from matrix-synapse"
    ]
  },
  "psi4": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2385428",
      "https://bugzilla.redhat.com/show_bug.cgi?id=2371776"
    ],
    "summaries": [
      "psi4: FTBFS in Fedora rawhide/f43",
      "F43FailsToInstall: psi4"
    ]
  },
  "python-dataclassy": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2325200"
    ],
    "summaries": [
      "python-dataclassy fails to build with Python 3.14: TypeError: Beta() takes no arguments"
    ]
  },
  "python-ffc": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2385467",
      "https://bugzilla.redhat.com/show_bug.cgi?id=2371902"
    ],
    "summaries": [
      "python-ffc: FTBFS in Fedora rawhide/f43",
      "F43FailsToInstall: python3-ffc"
    ]
  },
  "python-optking": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2385505",
      "https://bugzilla.redhat.com/show_bug.cgi?id=2372017"
    ],
    "summaries": [
      "python-optking: FTBFS in Fedora rawhide/f43",
      "F43FailsToInstall: python3-optking"
    ]
  },
  "python-pamela": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2366474"
    ],
    "summaries": [
      "python-pamela fails to build with Python 3.14: pamela.PAMError: [PAM Error 26] Critical error - immediate abort"
    ]
  },
  "python-qcengine": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2385529",
      "https://bugzilla.redhat.com/show_bug.cgi?id=2372090"
    ],
    "summaries": [
      "python-qcengine: FTBFS in Fedora rawhide/f43",
      "F43FailsToInstall: python3-qcengine"
    ]
  },
  "zeek": {
    "urls": [
      "https://bugzilla.redhat.com/show_bug.cgi?id=2366467"
    ],
    "summaries": [
      "zeek fails to build in Fedora Rawhide: Could not find prerequisite package 'BIND'"
    ]
  }
}